misk-tailwind/src/main/kotlin/misk/tailwind/pages/Navbar.kt

```diff
@@ -0,0 +1,228 @@
+          div("hidden flex grow flex-col gap-y-5 overflow-y-auto bg-gray-900 px-6 ring-1 ring-white/10") {
+            attributes["data-toggle-target"] = "toggleable"
+            attributes["data-css-class"] = "hidden"
+
+            div("flex h-16 shrink-0 items-center") {
+              if (menuSections.isNotEmpty()) {
+                button(classes = "-m-2.5 p-2.5 text-white xl:hidden") {
+                  type = ButtonType.button
+                  attributes["data-action"] = "toggle#toggle"
+                  span("sr-only") { +"""Close sidebar""" }
+                  // TODO add to Heroicons
+                  unsafe {
+                    raw(
+                      """
+                  <svg class="h-6 w-6 text-white" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
+                    <path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12" />
+                  </svg>
+                  """.trimIndent()
+                    )
+                  }
+                }
+              }
+            }
+//            div("flex h-16 shrink-0 items-center") {
+//              unsafe {
+//                raw(
+//                  """
+//                    <svg class="h-6 w-6 text-white" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
+//                      <path fill-rule="evenodd" d="M2 4.75A.75.75 0 012.75 4h14.5a.75.75 0 010 1.5H2.75A.75.75 0 012 4.75zM2 10a.75.75 0 01.75-.75h14.5a.75.75 0 010 1.5H2.75A.75.75 0 012 10zm0 5.25a.75.75 0 01.75-.75h14.5a.75.75 0 010 1.5H2.75a.75.75 0 01-.75-.75z" clip-rule="evenodd" />
+//                    </svg>
+//                    """.trimIndent()
+//                )
+//              }
+//            }
+            NavMenu(menuSections)
+          }
+        }
+      }
+    }
+    // +"""<!-- Static sidebar for desktop -->"""
+    div("hidden xl:fixed xl:inset-y-0 xl:z-50 xl:flex xl:w-72 xl:flex-col") {
+      // +"""<!-- Sidebar component, swap this element with another sidebar if you like -->"""
+      div("flex grow flex-col gap-y-5 overflow-y-auto bg-black/10 px-6 ring-1 ring-white/5") {
+        div("flex h-16 shrink-0 items-center") {
+          a(homeHref) {
+            img(classes = "h-8 w-auto") {
+              src = "/static/favicon.ico"
+              alt = "Dashboard Logo"
+            }
+          }
+        }
+        NavMenu(menuSections)
+      }
+    }
+    div("xl:pl-72") {
+      // +"""<!-- Sticky search header -->"""
+      div("sticky top-0 z-40 flex h-16 shrink-0 items-center gap-x-6 border-b-4 border-b-${deployment.asColor()} bg-gray-900 px-6 shadow-sm sm:px-6 lg:px-6") {
+        if (menuSections.isNotEmpty()) {
+          button(classes = "-m-2.5 p-2.5 text-white xl:hidden") {
+            attributes["data-action"] = "toggle#toggle"
+            type = ButtonType.button
+            span("sr-only") { +"""Open sidebar""" }
+            // TODO add to Heroicons
```

Comment:
Anything blocking this?

Reply:
I've removed the TODOs because I'm not sure if they're actually part of that icon set.